crates/ruff_python_resolver/src/lib.rs

```diff
@@ -129,6 +128,14 @@ mod tests {
         env_logger::builder().is_test(true).try_init().ok();
     }
 
+    macro_rules! assert_debug_snapshot_normalize_paths {
+        ($value: ident) => {
+            // The debug representation for the backslash are two backslashes (escaping)
+            let $value = format!("{:#?}", $value).replace("\\\\", "/");
+            insta::assert_display_snapshot!($value);
```

Comment:
We shouldn't need to call into the `Display` implementation, considering that `$value` already is a string

```suggestion
            insta::assert_snapshot!($value);
```

Reply:
in this case it uses the debug representation of the string, which contains literal `\n`; i'll added comment